pyERGM/metrics.py

```diff
@@ -1169,3 +1172,27 @@ def get_ignored_features(self):
             parameter_names += metric._get_ignored_features()
 
         return parameter_names
+
+
+class SumDistancesConnectedNeurons(Metric):
+    """
+    This function calculates the sum of euclidean distances between all pairs of connected neurons.
+    rows = samples, columns = axes
+    """
+
+    def __init__(self, positions):
+        super().__init__(requires_graph=False)
+        self._is_directed = True
```

Comment:
True. I changed it so it would be an input from the user like in the NodeAttrSum function.

Reply:
Great, but the calculation is also a bit different for undirected graphs... Our convention is that there are 2 different classes for directed and undirected graphs, and the `calculate` of each one of them is implemented accordingly (see  `NumberOfEdges` and `Degree` classes). I think this can be a good solution here too - specify already in the name of the class whether it expects directed or undirected graphs, and implement `calculate` accordingly (it's OK to stay only with the directed version for now).